core/interop/pinvoke/marshaling/src/MarshalingSample/MarshalingSample.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <OutputType>exe</OutputType>
+    <UseAppHost>true</UseAppHost>
```

Comment:
Just curious why is this necessary - I thought that "exe" projects always have `UseAppHost` set to true (for 3.0 and above).

Reply:
Not on OSX

Reply:
err yes that's what I meant ;)